### DISCUSSION TITLE
Add exif ifd's to photo info

### DISCUSSION THREAD

**Discussion by ippocratis** [savvasdalkitsis/uhuruphotos-android#92] [Ideas]
I dont even know if this is possible as librephotos frontend does not display any metadata appart from timestamp and map but looks like it reads them?

https://github.com/LibrePhotos/librephotos/issues/10#issuecomment-752735540
And
https://github.com/LibrePhotos/librephotos/issues/77#issuecomment-753404993

Anyway maybe its beyond the scope of the app but I'd love to see that

Thanks

**Comment by savvasdalkitsis** [comment]
I could have a look to see about reading the exif data in the full image response. Librephotos doesn't seem to currently export the data as api responses but if the server serves the original unmodified file when requesting the full resolution, i should be able to extract the data locally. Will have a look at this
